src/pudl/metadata/resources/eia860.py

```diff
@@ -232,6 +232,9 @@
                     "generation_eia923",
                     "denorm_generation_eia923",
                     "denorm_generation_monthly_eia923",
+                    "generation_fuel_by_generator_energy_source_monthly_eia923",
+                    "generation_fuel_by_generator_monthly_eia923",
+                    # "generation_fuel_by_generator_energy_source_owner_monthly_eia923",
```

Comment:
should you take out all of these #commented out monthly tables we probs aren't going to make?

Reply:
Took out the commented out table.